ui/ui-app/init-dev.sh

```diff
@@ -8,7 +8,7 @@ CONFIG_TYPE=$1
 
 if [ "x$CONFIG_TYPE" = "x" ]
 then
-  CONFIG_TYPE="none"
+  CONFIG_TYPE="local"
```

Comment:
Should we add a note in the readme about this env var? (I'm going to forget about it pretty quickly 😅 )

Reply:
This was already documented in the `ui-app` readme, but that readme DID need to be updated, so good call and I have done so.  :)